lib/Cake/TestSuite/CakeTestLoader.php

```diff
@@ -31,7 +31,7 @@ class CakeTestLoader extends PHPUnit_Runner_StandardTestSuiteLoader {
  * Load a file and find the first test case / suite in that file.
  *
  * @param string $filePath The file path to load
- * @param string $params Additional parameters
+ * @param array $params Additional parameters
  * @return ReflectionClass
  */
 	public function load($filePath, $params = '') {
```

Comment:
But then the default value wouldnt make sense. Maybe instead of changing the docs you want to add a cast somewhere.

Reply:
Looks like these type hints and `_basePath` hints are wrong. Looks like both should be `array|string`.